test/fixtures/mock-registry.js

```diff
@@ -74,12 +74,39 @@ class MockRegistry {
       team = team.slice(1)
     }
     const [scope, teamName] = team.split(':')
-    this.nock.delete(
+    this.nock = this.nock.delete(
       `/-/team/${encodeURIComponent(scope)}/${encodeURIComponent(teamName)}/package`,
       { package: spec }
     ).reply(200)
   }
 
+  couchlogin ({ username, password, email, otp, token = 'npm_default-test-token' }) {
+    this.nock = this.nock
+      .post('/-/v1/login').reply(401, { error: 'You must be logged in to publish packages.' })
+    if (otp) {
+      // TODO otp failure results in a 401 with
+      // {"ok":false,"error":"failed to authenticate: Could not authenticate ${username}: bad otp"}
+    }
```

Comment:
Just confirming that this was meant as a future TODO and not one for this PR.

Reply:
Yes. the existing tests didn't test adduser w/ otp.

Reply:
I just left the comment in there cause I had recorded the requests and wanted to store it for the future.